tests/test_model.py

```diff
@@ -0,0 +1,110 @@
+from ocetrac.model import Tracker 
+
+import pytest
+import unittest
```

Comment:
I am a bit confused here. You are using both pytest and unittest simultaneously? Is there a reason to do this?

Reply:
Oops! No reason. I will get rid of unittest.